api/application.py

```diff
@@ -3,8 +3,8 @@
 
 import subprocess
 
-app = Flask(__name__)
-CORS(app)
+application = Flask(__name__)
```

Comment:
I saw `@app` used basically everywhere online, including Flask's documentation.
https://flask.palletsprojects.com/en/1.1.x/appcontext/

Was this part of the required renaming you mention in the main PR comment?

Reply:
Yes, EB requires the file to be named application.py and the application variable to be named application. There's a way to use a "non-standard" filename but it seemed better to go with their defaults.

Reply:
I had to write all the .ebextensions, hence why they are being checked in.

There's another .elasticbeanstalk directory that has other kinds of one-time/prepopulated settings that isn't being committed.